plugin-architecture/plugins/pipenv-python-script-plugin/src/Main.java

```diff
@@ -0,0 +1,36 @@
+package src;
+public class Main {
+    public boolean PreExecute() {
+        System.out.println("PreExecute: pipenv-python-script-plugin");
+        return true;
+    }
+    public boolean Execute() {
+        System.out.println("Execute: pipenv-python-script-plugin");
```

Comment:
Should we go with the shared object / dylib approach here ? Since the shared library is also calling the system python command, we could have this java class, install the venv dependencies during `PreExecute` and call the python script during `Execute` instead. 

Loading a shared object / dylib can be example of another plugin. In this case directly calling python from java would suffice i think.